it/controllers/SignUpControllerISpec.scala

```diff
@@ -24,8 +24,11 @@ class SignUpControllerISpec extends ComponentSpecBase {
 
       val res = IncomeTaxSubscription.signUp(testNino)
 
+      println(res)
```

Comment:
This is the bit I got stuck on for ages - the jsonBodyAs test throws an exception, which crashes scalatest.  Simply separating the two tests allowed the error to be seen.